Packs/ExtFilter/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+##### Test - ExtFilter Main
+- Modified task titles
+
+#### Scripts
+##### ExtFilter
+- Added `email-header: decode`
```

Comment:
I think we can improve the release note entry here to mention that this is an operation we added here

Reply:
Added it.